code/controllers/subsystem/starmap.dm

```diff
@@ -149,6 +179,22 @@ SUBSYSTEM_DEF(starmap)
 
 		from_time = 0
 		to_time = 0
+
+		in_transit_planet = 0
```

Comment:
this is already done on line 202 and 203. If you need to move it up here, remove the old one.